core/src/scala/com/mygdx/game/screen/TiledScreen.scala

```diff
@@ -0,0 +1,51 @@
+package com.mygdx.game.screen;
+
+import com.badlogic.gdx.maps.tiled._;
+import com.badlogic.gdx.{ApplicationAdapter, Gdx, Input, InputProcessor, ScreenAdapter};
+import com.badlogic.gdx.graphics.{GL20, OrthographicCamera, Texture};
+import com.badlogic.gdx.graphics.g2d._;
+import com.badlogic.gdx.maps.tiled.renderers.OrthogonalTiledMapRenderer;
+
+import com.mygdx.game.GameApplication
+
+class TiledScreen(game: GameApplication) extends ScreenAdapter{
+
+    var map : TiledMap = _;
```

Comment:
Good thinking, I didn't realize delayed initialization could be removed, but you're right, I never need to change map or renderer after they're initialized, so it makes more sense to just do so as soon as possible.